jenkins/spark-premerge-build.sh

```diff
@@ -37,7 +37,7 @@ export PATH="$SPARK_HOME/bin:$SPARK_HOME/sbin:$PATH"
 tar zxf $SPARK_HOME.tgz -C $ARTF_ROOT && \
     rm -f $SPARK_HOME.tgz
 
-mvn -U -B $MVN_URM_MIRROR '-Pinclude-databricks,!snapshot-shims' clean verify -Dpytest.TEST_TAGS=''
```

Comment:
Will there be a follow on to enable databricks?

Reply:
we don't really need it in the premerge because we don't test there. The nightly build and deploy builds still have it enabled.

Reply:
ie we don't test databricks in premerge

Reply:
We don't test it, but don't we at least try to _compile_ against it in premerge?

Reply:
There is nothing to compile against unless we can ssh into the databricks cluster, which has not been setup yet.